plugins/builds/artifacts/get.js

```diff
@@ -17,11 +17,11 @@ module.exports = config => ({
         tags: ['api', 'builds', 'artifacts'],
         auth: {
             strategies: ['session', 'token'],
-            scope: ['user']
+            scope: ['user', 'build']
```

Comment:
I think something went wrong with last commit, was supposed to add `session` back here based on discussion https://github.com/screwdriver-cd/screwdriver/pull/919#discussion_r174239529

Reply:
Not to scope, but to strategies (which it is there).

Reply:
Builds _should_ be able to access this endpoint, no?

Reply:
aah sorry I got confused. Build does not call this `get` endpoint, so don't think it's required.